examples/covariance/plot_robust_vs_empirical_covariance.py

```diff
@@ -68,6 +68,7 @@
 range_n_outliers = np.concatenate(
     (np.linspace(0, n_samples / 8, 5),
      np.linspace(n_samples / 8, n_samples / 2, 5)[1:-1]))
+range_n_outliers = np.array(range_n_outliers, dtype=np.int)
```

Comment:
Use `.astype(np.int)` on the previous array (you can do everything in a single statement)